src/App.js

```diff
@@ -37,8 +37,11 @@ function SMLinks(props) {
 
 function SMLinkNav(props) {
   let classes = "";
-  if(props.menunav==='true') {
-    classes+= "lg:hidden";
+  if(props.menunav==='true') {//checking if the sm links are in the main nav or in the slide in nav
```

Comment:
This could be simplified a bit to:

```jsx
// hide on desktop if `menunav=true` (i.e. rendered from the slide in nav)
const classNames = props.menunav ? 'lg:hidden' : 'lg:block'
```

I wonder if it might be cleaner if this component isn't aware of the different type of navs and instead just takes a prop like `hideOnDesktop`. Then, when rendering from the main nav, you could pass `<SMLinkNav hideOnDesktop={false} />` and `<SMLinkNav hideOnDesktop={true} />` from the slide in nav.